src/nix/eval.cc

```diff
@@ -87,8 +89,8 @@ struct CmdEval : MixJSON, InstallableValueCommand, MixReadOnlyOption
                     // FIXME: disallow strings with contexts?
                     writeFile(path.string(), v.string_view());
                 else if (v.type() == nAttrs) {
-                    // TODO abstract mkdir perms for Windows
-                    createDir(path.string(), 0777);
+                    // N.B. this is idempotent
+                    fs::create_directory(path.string());
```

Comment:
Is this supposed to be there still?

Reply:
It is a fine change change, but the other one is not.